ledger/apptxn_test.go

```diff
@@ -2156,6 +2156,8 @@ func TestInnerCreatedAppsAreCallable(t *testing.T) {
 	t.Parallel()
 
 	genBalances, addrs, _ := ledgertesting.NewTestGenesis()
+	l := newTestLedger(t, genBalances)
+	defer l.Close()
```

Comment:
I am not too familiar with testing tools in ledger, but this seems like a reasonable addition by context... not important to this PR I guess

Reply:
I'm going to remove it.  It looks entirely extraneous. Thanks.